docs/about/release-notes.rst

```diff
@@ -22,6 +22,7 @@ Features
   * Support unit conversion to energy transfer, for inelastic TOF experiments `#1635 <https://github.com/scipp/scipp/pull/1635>`_.
   * Support loading/converting Mantid ``WorkspaceGroup``, this will produce a ``dict`` of data arrays `#1654 <https://github.com/scipp/scipp/pull/1654>`_.
   * Fixes to support loading/converting ``McStasNexus`` files `#1659 <https://github.com/scipp/scipp/pull/1659>`_.
+* `is_close` added (`is_approx` deprecated). Fuzzy data comparison is_close is an analogue to numpy's `isclose` 
```

Comment:
I don't think we need a deprecation process right now? Just remove `is_approx`? Also, as said above, shouldn't we call is `isclose` to minimize confusion?

Reply:
Yes I think `isclose` is what we would want so we should name it this first time round. I'll do a follow up to rename the others.